pkg/api/unversioned.go

```diff
@@ -22,16 +22,35 @@ import (
 
 // This file contains API types that are unversioned.
 
-// APIVersions lists the api versions that are available, to allow
-// version negotiation. APIVersions isn't just an unnamed array of
-// strings in order to allow for future evolution, though unversioned
+// APIVersions lists the groups and versions that are available, to allow
+// clients to discover the API. APIVersions isn't just an unnamed array of
+// strings in order to allow for future evolution, though unversioned.
```

Comment:
Maybe remove "APIVersions isn't just an unnamed array of strings in order to allow for future evolution, though unversioned."

It is very confusing.